aws cooc to xsiam and xsoar (#44379)
demisto/content: Packs/ApiModules/ReleaseNotes/2_4_17.md

#### Scripts

##### COOCApiModule

> - Fixed a bug in `is_gov_account` where the `cloud_partition` check block was missing, causing the function to return `None` instead of `True`/`False`.

Reviewer: ```suggestion
- Fixed an issue in the `is_gov_account` function where the `cloud_partition` check block was missing, causing it to return `None` instead of `True`/`False`.
```
